src/shared/containers/ResourceEditor.tsx

```diff
@@ -119,23 +123,27 @@ const ResourceEditorContainer: React.FunctionComponent<{
       }
     )) as Resource;
     const orgProject = getOrgAndProjectFromResourceObject(data);
-    dispatch(
-      InitNewVisitDataExplorerGraphView({
-        current: {
-          _self: data._self,
-          types: getNormalizedTypes(data['@type']),
-          title: getResourceLabel(data),
-          resource: [
-            orgProject?.orgLabel ?? '',
-            orgProject?.projectLabel ?? '',
-            data['@id'],
-            data._rev,
-          ],
-        },
-        limited: true,
-      })
-    );
-    navigate.push('/data-explorer/graph-flow');
+    if (location.pathname === '/data-explorer/graph-flow') {
+      dispatch(InitDataExplorerGraphFlowLimitedVersion(true));
+    } else {
+      dispatch(
+        InitNewVisitDataExplorerGraphView({
+          current: {
+            _self: data._self,
+            types: getNormalizedTypes(data['@type']),
+            title: getResourceLabel(data),
+            resource: [
+              orgProject?.orgLabel ?? '',
```

Comment:
It would be nice to have a utility that returns this resource array. I can foresee cases where we accidentally changes the sequence of elements (for example, project before organization).